tests/src/Interop/IJW/ManagedCallingNative/ManagedCallingNative.csproj

```diff
@@ -24,6 +24,9 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
   </PropertyGroup>
+  <PropertyGroup>
+    <CopyDebugRuntimeDllsToOutputDirectory Condition="('$(Configuration)' == 'Debug') Or ('$(Configuration)' == 'Checked')">true</CopyDebugRuntimeDllsToOutputDirectory>
```

Comment:
Booo. Any property that has the word `Debug` in the name should _not_ require a conditional build type check. Please apply checking for `Debug` or `Checked` to the `ItemGroup` in the targets file. Setting this property should be a nop on retail builds.